segmentstore/server/src/main/java/io/pravega/segmentstore/server/tables/TableExtensionConfig.java

```diff
@@ -92,6 +93,12 @@
      */
     private final Duration compactionFrequency;
 
+    /**
+     * Determines whether to enable Table Compaction or not. Note that this will impact all Table Segments in the
```

Comment:
Does compaction include truncation?

Reply:
Not directly, but indirectly it is dictating truncation. `WriterTableProcessor` is the class doing the truncation, but it relies on `compactor.calculateTruncationOffset(highestCopiedOffset)` to perform it. My understanding is that this will influence the frequency of truncations for sure (the segment can still be truncated, but it will happen less frequently according to my understanding).